force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -217,17 +242,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return A List of integers from 1 to 'n'.
      */
     public static List<Integer> createIntegerList(Integer n) {
-        // Create a new list of integers 
-
-        /*
-        for() {
-            //add integer to the list
+        List<Integer> numberList = new List<Integer>();
+        for(Integer i = 1; i <= n; i++){
+            numberlist.add(i);
```

Comment:
Good work.  The normal developer convention is to start at 0 and use I < n.. but this works just as well.